rake_nltk/rake.py

```diff
@@ -166,20 +166,12 @@ def _generate_phrases(self, sentences):
            @return: List of List of strings where each sublist is a collection
```

Comment:
Can you update the @return string to
`@return: List of string tuples where each tuple is a collection of words, which form a contender phrase.`
so that code change and corresponding documentation are in the same commit.